factor_analyzer/factor_analyzer.py

```diff
@@ -253,18 +253,15 @@ def __init__(self,
 
         rotation = rotation.lower() if isinstance(rotation, str) else rotation
         if rotation not in POSSIBLE_ROTATIONS + [None]:
-            raise ValueError('The rotation must be None, or in the following set: '
-                             '\{{}\}'.format(', '.join(POSSIBLE_ROTATIONS)))
+            raise ValueError(f"The rotation must be None, or in the following set: {', '.join(POSSIBLE_ROTATIONS)}")
```

Comment:
Hmm actually, I am not sure we even need the join? Why not just?

```suggestion
            raise ValueError(f"The rotation must be None, or one of the following: {POSSIBLE_ROTATIONS}")
```

What do you think @hotessy ?

Reply:
@desilinguist, if we're going for a list, how about:
```python
raise ValueError(f"The rotation must be one of the following: {POSSIBLE_ROTATIONS + [None]}")
```

The syntax will also match `if` condition used to enter this chunk.